study.md

```diff
@@ -18,17 +18,19 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works?
 
 ```md
-<!-- your answer here -->
+<!-- An event loop is whe n you have a callback quene heap and other objects and functions
+that are all running together in a loop -->
 ```
 
 In your own words, describe the difference between sync and async:
 
 ```md
-<!-- your answer here -->
+<!-- Sync mean at the same time and async means it happens at different time. -->
```

Comment:
Sync is one after another, Async is while one function is waiting to finish, something else can run.